docker/docker_tools.py

```diff
@@ -248,12 +276,23 @@ def build(
             print(f"Unable to run git: {e} Installing...")
             xqt("sudo apt-get install -y --no-install-recommends git")
 
+        # Check if Clone-All Flag is set
+        if clone_all != "RunestoneInteractive":
+            clone_bs = clone_all
+            clone_rc = clone_all
+            clone_rs = clone_all
+
         # Are we inside the Runestone repo?
         if not (wd / "nginx").is_dir():
             change_dir = True
             # No, we must be running from a downloaded script. Clone the runestone repo.
             print("Didn't find the runestone repo. Cloning...")
-            xqt("git clone https://github.com/RunestoneInteractive/RunestoneServer.git")
+            try:
+                # Check if possible to clone RunestoneServer with Custom Repo
+                xqt("export GIT_TERMINAL_PROMPT=0 && git clone https://github.com/%s/RunestoneServer.git"%clone_rs)
```

Comment:
You can use `xqt("GIT_TERMINAL_PROMPT=0...` instead (no need to export).

Reply:
Also, use an f-string instead -- that's easier to read.

Reply:
Must have forgot f-string is a thing again... Anyway, I changed them to f-strings like you suggested and it appears that export is necessary to temporarily disable the Git Prompt to pass-though the error. I attempted to execute it without ```export``` but it would act as if it was never executed. I did noticed that the export only affects that session of script execution so it isn't exporting to the host system.

Reply:
OK, sounds good.